Project.toml

```diff
@@ -10,6 +10,8 @@ Libdl = "8f399da3-3557-5675-b5ff-fb832c97cbdb"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 Markdown = "d6f4376e-aef5-505a-96c1-9c027394607a"
 Polyhedra = "67491407-f73d-577b-9b50-8179a7c68029"
+SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
+StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
```

Comment:
Why adding these ?

Reply:
Tests are erroring without, I can move them to tests.
It seems that since we are injecting the tests from Polyhedra, we need the test dependencies in this repo too